lua/cli_flash.lua

```diff
@@ -349,7 +349,14 @@ function detect_chiptype(aArgs)
 			print("")
 			printf("Chip type: (%d) %s", iChiptype, strChipName)
 			print("")
-			fOk = true
+

```

Comment:
Please fix the indentation. (I know, the whole file is incorrectly indented with tabs... sorry) 
Alternatively, the block could be removed and line 341 changed to
if iChiptype ~= nil and iChiptype ~= romloader.ROMLOADER_CHIPTYP_UNKNOWN then...